libs/ojsonnet/Eval_jsonnet.ml

```diff
@@ -102,63 +76,73 @@ let log_call env str tk =
 (* Builtins *)
 (*****************************************************************************)
 (* alt: could move to Value_jsonnet.ml *)
-let std_type _env (v : V.value_) : string =
+let std_type _env (v : value_) : string =
   match v with
-  | V.Primitive (V.Null _) -> "null"
-  | V.Primitive (V.Bool _) -> "boolean"
-  | V.Primitive (V.Double _) -> "number"
-  | V.Primitive (V.Str _) -> "string"
-  | V.Object _ -> "object"
-  | V.Array _ -> "array"
-  | V.Function _ -> "function"
-
-let std_primivite_equals _env (v : V.value_) (v' : V.value_) : bool =
+  | Primitive (Null _) -> "null"
+  | Primitive (Bool _) -> "boolean"
+  | Primitive (Double _) -> "number"
+  | Primitive (Str _) -> "string"
+  | ObjectVal _ -> "object"
```

Comment:
See, because of your open you had to rename some constructors, but if instead you qualify, there's no need to rename